prospero-cli/src/main/java/org/wildfly/prospero/cli/commands/CloneCommand.java

```diff
@@ -0,0 +1,125 @@
+/*
+ * Copyright 2022 Red Hat, Inc. and/or its affiliates
+ * and other contributors as indicated by the @author tags.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.wildfly.prospero.cli.commands;
+
+import org.wildfly.prospero.Messages;
+import org.wildfly.prospero.actions.Console;
+import org.wildfly.prospero.actions.InstallationExportAction;
+import org.wildfly.prospero.actions.InstallationRestoreAction;
+import org.wildfly.prospero.cli.ActionFactory;
+import org.wildfly.prospero.cli.RepositoryDefinition;
+import org.wildfly.prospero.cli.ReturnCodes;
+import org.wildfly.prospero.cli.commands.options.LocalRepoOptions;
+import org.wildfly.prospero.wfchannel.MavenSessionManager;
+import picocli.CommandLine;
+
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+
+@CommandLine.Command(name = CliConstants.Commands.CLONE)
+public class CloneCommand extends AbstractCommand {
+
+    @CommandLine.Spec
+    protected CommandLine.Model.CommandSpec spec;
+
+    public CloneCommand(Console console, ActionFactory actionFactory) {
+        super(console, actionFactory);
+    }
+
+    public void addSubCommands(CommandLine rootCmd) {
+        CommandLine cloneCmd = rootCmd.getSubcommands().get(CliConstants.Commands.CLONE);
+        cloneCmd.addSubcommand(new CloneExportCommand(console, actionFactory))
+          .addSubcommand(new CloneImportCommand(console, actionFactory));
+    }
+
+    @Override
+    public Integer call() {
+        spec.commandLine().usage(console.getErrOut());
+        return ReturnCodes.INVALID_ARGUMENTS;
+    }
+
+    @CommandLine.Command(name = CliConstants.Commands.EXPORT)
+    private static class CloneExportCommand extends AbstractCommand {
+
+        @CommandLine.Option(names = CliConstants.DIR, order = 1)
+        Optional<Path> directory;
+
+        @CommandLine.Option(names = CliConstants.ARG_PATH, required = true, paramLabel = CliConstants.PATH, order = 2)
+        String outPath;
```

Comment:
Is there a reason that's a String rather then a Path?

Reply:
Because `InstallationExportAction.export(String exportName)` is using `String`, I think we can change the export to use `Path` as the parameter, WDYT?

Reply:
Ah, yes please, let's change `InstallationExportAction.export` 👍